### DISCUSSION TITLE
Can the graph show nodes in both directions?

### DISCUSSION THREAD

**Discussion by cparsell** [oleeskild/obsidian-digital-garden#378] [Ideas]
I tried to check if this had been addressed anywhere but didn't see it. Forgive me if I didn't look hard enough.

I've noticed that when I click a link (equivalent of one level deeper on the graph) the next local graph does not show where I came from.  
If I start at Home, and click on a link to go to Subpage 1, the local graph (on Subpage 1) does not show Home as a connection point. It only shows Sub-sub-pages further down the graph. It seems the graph only shows nodes in one direction. 

Obsidian's graph shows links in both directions. Is it possible to add this as a feature or an option?

I can't quite understand the logic in the graph component or I'd propose a solution.

**Comment by oleeskild** [comment]
This sounds like a bug, it should show in both directions. Do you have a link to your garden?

**Reply by cparsell** [reply]
Sure. Here is a [link](https://cparsell.vercel.app).

**Reply by oleeskild** [reply]
Are you on the latest version of the template? (Ref: https://dg-docs.ole.dev/getting-started/06-updating-the-template/#updating-the-template) ?

**Reply by cparsell** [reply]
I should be. I installed it last week. I wondered if it was something only I experienced so I checked one of the "sites other people have created" ([this one](https://hermitage.utsob.me/)) and it looked like that one behaved that way too.
